res/xml/devanagari.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard>
+  <row>
+    <key shift="0.35" width="0.9" key0="क" key1="ख" key2="घ" key3="ङ"  key4="ग"/>
```

Comment:
The biggest difference between my layout and the current Hindi layout is the following:

The devanagari characters are organized by mouth/tongue movements https://omniglot.com/writing/devanagari.htm . So for example ग=g, क=k (these are similar tongue movements). I've grouped all similar-tongue-movement characters into the same key

Reply:
I talk to one of my hindi-speaking friends, and he said that this grouping (by tongue movements) is the best way to organize the alphabet (and thus also a keyboard layout)